x/staking/keeper/slash.go

```diff
@@ -8,7 +8,7 @@ import (
 	"cosmossdk.io/math"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
-	"github.com/cosmos/cosmos-sdk/x/staking/types"
+	types "github.com/cosmos/cosmos-sdk/x/staking/types"
```

Comment:
can't it be left as `types`?

Reply:
This will technically get us closer to the original Cosmos SDK codebase, so IMO no need to undo this change.